README.md

```diff
@@ -885,7 +885,7 @@ metadata:
   name: fluentd-only-stdout
   namespace: fluent
   labels:
-    output.fluentd.fluent.io/enabled: "true"
+    output.fluentd.fluent.io/mode: "true"
```

Comment:
`output.fluentd.fluent.io/mode: "true"` should be changed to `filter.fluentd.fluent.io/mode: "fluentd-only"
` for both the selector and output labels